.github/workflows/dummy.yml

```diff
@@ -0,0 +1,30 @@
+name: Add a Dummy commit to trigger build
+
+on:
+  push:
+    branches: master
+    paths-ignore:
+      - .github/workflows/dummy.yml
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Perhaps better at 07:30am every day, just so that if builds fail we start the day by fixing the issue first?

Reply:
ok